jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -202,6 +193,45 @@ private void uriMismatchWarningCheck(IType type, List<Diagnostic> diagnostics, I
         }
     }
 
+    /**
+     * Create a warning diagnostic if a ServerEndpoint annotation does not follow the rules.
```

Comment:
You are not creating a warning diagnostic. You are creating an error diagnostic. Did you mean to do it only a warning? If not, could you update the comments and the respective methods/variables that refer to it as a warning as this could lead to a confusion?

Reply:
It was a typo, meant to be an error.